.circleci/config.yml

```diff
@@ -80,15 +80,15 @@ jobs:
             - ~/specs-repo
   install_pyspec_test:
     docker:
-      - image: ethpandaops/circleci-python-rust:latest
+      - image: cimg/python:3.12.4
```

Comment:
@parithosh could you help confirm it?

Reply:
For reference, I pinged @kevaundray and asked him to publish pre-built wheels for Python 3.12 to py_arkworks. You can see these here: https://pypi.org/project/py-arkworks-bls12381/#files

Reply:
Ah amazing! I only added rust to get around the `py_arkworks_bls12381`. If that has been fixed, i agree we should switch back to the official python image.